books.js

```diff
@@ -0,0 +1,70 @@
+const bookSection = document.querySelector('.books');
+const bookTitle = document.getElementById('book-title');
+const bookAuthor = document.getElementById('book-author');
+const form = document.getElementById('form');
+
+function getBooks() {
+  let myBooks;
+  if (localStorage.getItem('newBooks') === null) {
+    myBooks = [];
+  } else {
+    myBooks = JSON.parse(localStorage.getItem('newBooks'));
+  }
+  return myBooks;
+}
+
+function showBooks() {
+  bookSection.innerHTML = '';
+  const myBooks = getBooks();
+  for (let i = 0; i < myBooks.length; i += 1) {
+    const book = myBooks[i];
+    const bookList = `
+    <div id="book-info">
+    <p>${book.title}</p>
```

Comment:
- [ ] Kindly change all regular functions to es6 arrow functions. As stated in the [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.).